etcdserver/v3_server.go

```diff
@@ -260,7 +260,11 @@ func (s *EtcdServer) LeaseRenew(ctx context.Context, id lease.LeaseID) (int64, e
 			}
 		}
 	}
-	return -1, ErrTimeout
+
+	if cctx.Err() == context.DeadlineExceeded {
```

Comment:
Do we need to change the return logic in `lease timetolive and renew`?

Reply:
cctx.Err() can be either due to timeout or cancellation. I think in the original code, user can cancel the context but the returned error is ErrTimeout.